README.md

```diff
@@ -1,26 +1,102 @@
-
-### Filtering
-Map data can be filtered by specifying a query in the form `( QUERY ) as VARIABLE` as the value of `table` for a layer in `map-config.mml`. Different combinations of layers can be fetched by adding `?layers=layer1,layer2,layer...N` as a query string to a tile or UTF grid URL. Make sure that all necessary columns are present in the query. For example, if your geometry column is called `geom`, and you want to display a UTF grid of content from the column `name` filtered by the content of the column `type`, your query would look something like `SELECT geom,name,type FROM table WHERE type='something'`. If no layers are specified in the query string, all layers are displayed in the same tile.
-
-### Debugging
- The local development server exposes a websocket to a node inspector that can be attached to your IDE of choice to step through and debug your code. Here are instructions on how to do so using Google Chrome's Dev Tools:
-  * Start the development server with `./scripts/server`.
-  * Open Google Chrome and navigate to `about:inspect`.
-  * There should be an option listed as something along the lines of "Target (v8.10.0)" with the Node.js logo and a path like `file:///home/tiler/node_modules/claudia-local-api/bin/claudia-local-api`. Click "inspect", underneath.
- * A new window will open up. Type `ctrl+p` to open a search bar that lets you navigate to your source code. **The transpiled code in `tiler/bin/` is what is actually being tracked by the debugger,**  not the source code in `tiler/src/`.
+#### Datasources
+Tilegarden supports the use of any geospatial data source that Mapnik/Carto does (shapefile, postgis, pgraster, raster). However, bear in mind that only PostGIS data sources support custom queries, and are thus the only ones that allow you to perform additional filtering on your data (see [Filters](#filters)). Also, attempting to bundle large local data files into your Tilegarden deployment could lead to rejection by AWS Lambda due to size restrictions.
+
+#### Map Styles
+[Map styles are specified in CartoCSS, the specification for which can be found here.](https://cartocss.readthedocs.io/en/latest/) The default stylesheet is located at `src/tiler/src/config/style.mss`. Multiple stylesheets can be used by adding them to the parameter “Stylesheet” in `src/tiler/src/config/map-config.mml`. Each .mss “class” refers to a map layer (its “id” property), specified in `map-config.mml`. 
+
+#### Filters
+Filters are specified by altering the query of one of your map’s layers in `src/tiler/src/config/style.mss`, and can then be fetched through the API. To create a filtered layer, modify the “table” property of the layer to have the format `(QUERY) as VARIABLE`. 
+   * Make sure each layer has unique “id” and “name” values.
```

Comment:
Is `name` required?  The demo ones are all identical to the `id`, so leaving it off (in the file and the instructions) would be preferable if it's allowed.

Reply:
That it a good question.